report_xlsx/models/res_currency.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResCurrency(models.Model):
+    _inherit = "res.currency"
+
+    def _get_format_amount(self):
```

Comment:
I'd add a flag to show the symbol or not
```suggestion
    def _get_format_amount(self, symbol=True):
```